### ISSUE TITLE
The Discord Link for Steamtinkerlaunch is Expired

### ISSUE THREAD

**Issue by ThisNekoGuy** [sonic2kk/steamtinkerlaunch#369]
![image](https://user-images.githubusercontent.com/28468662/144806166-44cee5c2-998f-4e58-8dc5-5be133361bb3.png)

(https://discord.gg/sE52kB7a)

Could the link to the server be updated?

**Comment by frostworx** [comment]
whoops, thanks. *just fixed it*
Looks like I only updated/created the discord shield on the top of the page and forgot this one.

**Comment by anon2828226** [comment]
where is this new/fixed link???????????????
Where the hell are any of these?

**Comment by frostworx** [comment]
*unsubscribing here* and greeting Terry Pratchett

**Comment by sonic2kk** [comment]
@anon2828226 There is no Discord link. This is an issue from over 30 months ago, there is no Discord, Reddit, or Matrix link. I do not wish to have involvement with SteamTinkerLaunch (or any development work) outside of GitHub.

The Discord server has been gone for a while I think, at least when I started contributing I did not hear of any Discord server and that was almost two months after this was closed. So the server has probably been gone for quite a long time. I also did not hear of any Matrix server, so likely there is a similar story there.

The subreddit was closed when I took over the project as I have no interest in maintaining a subreddit. We did ask if anyone had any interest in moderating the subreddit, back in October 2023 before closing it, but no one came forward (#928).

GitHub Discussions were re-opened briefly, but were not used correctly and were too stressful to manage, and have not been available for at least a year if not more.
